app/controllers/discussions_controller.rb

```diff
@@ -1,8 +1,9 @@
 class DiscussionsController < ApplicationController
 
-  before_action :set_event, only: [:index, :show, :new, :edit, :create, :update]
+  before_action :set_event, except: [:destroy]
```

Comment:
Sir Akhil sir told me to use except here instead of only.

On Fri, Jul 31, 2015 at 4:17 PM, Jaspreet Singh <notifications@github.com>
wrote:

> In app/controllers/discussions_controller.rb
> <https://github.com/SAkshat/iconf/pull/3#discussion_r35963696>:
>
> > @@ -1,8 +1,9 @@
> >  class DiscussionsController < ApplicationController
> >
> > -  before_action :set_event, only: [:index, :show, :new, :edit, :create, :update]
> > +  before_action :set_event, except: [:destroy]
>
> use only
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SAkshat/iconf/pull/3/files#r35963696>.
>



-- 

Akshat Singhal
Vinsol